tests/end_to_end/test_helper.cpp

```diff
@@ -522,6 +535,10 @@ GlueDeviceIoControl(
 
 _Requires_lock_not_held_(_fd_to_handle_mutex) int Glue_open_osfhandle(intptr_t os_file_handle, int flags)
```

Comment:
Missing `_Success_ == ...` SAL annotation?  (Other functions that have error injection added seem to have gained this SAL annotation as well, but not in this instance)